fastlane/actions/cordova_increment_version_number.rb

```diff
@@ -0,0 +1,88 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      CORDOVA_INCREMENT_VERSION_NUMBER_CUSTOM_VALUE = :CORDOVA_INCREMENT_VERSION_NUMBER_CUSTOM_VALUE
+    end
+
+    # To share this integration with the other fastlane users:
+    # - Fork https://github.com/KrauseFx/fastlane
+    # - Clone the forked repository
+    # - Move this integration into lib/fastlane/actions
+    # - Commit, push and submit the pull request
+
+    class CordovaIncrementVersionNumberAction < Action
+      def self.run(params)
+        # fastlane will take care of reading in the parameter and fetching the environment variable:
+        Helper.log.info "Parameter API Token: #{params[:api_token]}"
+
+        # sh "shellcommand ./path"
+
+        # Actions.lane_context[SharedValues::CORDOVA_INCREMENT_VERSION_NUMBER_CUSTOM_VALUE] = "my_val"
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.description
+        "A short description with <= 80 characters of what this action does"
+      end
+
+      def self.details
+        # Optional:
+        # this is your change to provide a more detailed description of this action
+        "You can use this action to do cool things..."
+      end
+
+      def self.available_options
+        # Define all options your action supports.
+
+        # Below a few examples
+        [
+          FastlaneCore::ConfigItem.new(key: :api_token,
+                                       env_name: "FL_CORDOVA_INCREMENT_VERSION_NUMBER_API_TOKEN", # The name of the environment variable
+                                       description: "API Token for CordovaIncrementVersionNumberAction", # a short description of this parameter
+                                       verify_block: proc do |value|
+                                          raise "No API token for CordovaIncrementVersionNumberAction given, pass using `api_token: 'token'`".red unless (value and not value.empty?)
```

Comment:
Line is too long. [181/100]<br>Use `&&` instead of `and`.<br>Use 2 (not 3) spaces for indentation.<br>Don't use parentheses around the condition of an `unless`.